drivers/overlay/overlay.go

```diff
@@ -271,6 +272,18 @@ func (d *Driver) getSupportsVolatile() (bool, error) {
 	return supportsVolatile, nil
 }
 
+func (d *Driver) getSupportsDataOnly() (bool, error) {
```

Comment:
nit: getSupportsDataOnly->supportsDataOnly

Reply:
we already have a `supportsDataOnly *bool` in the `Driver` struct